src/cljs/main/broadfcui/page/workspace/details.cljs

```diff
@@ -45,10 +45,12 @@
             :data-test-id "no-bucket-access"}
       (cond (= 404 bucket-status-code)
             (str "The Google bucket associated with this workspace"
-                 " does not exist. Please contact help@firecloud.org.")
+                 " does not exist. Please write our help forum for assistance"
+                 " https://gatkforums.broadinstitute.org/firecloud .")
```

Comment:
I attempted to make this and the other links below external links, but my clojurescript is not strong enough at this time, (and it's not the outlined solution in the jira issue). For my own knowledge, is it possible to inject an href into a str here? If so, how would I do it?
I attempted to use this line but it came out as a full string:
`(links/create-external {:href (config/forum-url)} "help forum")`

Reply:
`str` converts all of its arguments to strings and concatenates them together. You would have to create a DOM node:
```clojure
[:span {}
 "The Google... please contact "
 (links/create-external ...)
 " for assistance"]
```

Reply:
Actually, since the parent is already a div, you should put them in a list:
```clojure
(list
 "text"
 (links/create-external ...)
 "text")
```
The renderer knows to splice lists in, so `[:div {} (child1 child2)]` is the same as `[:div {} child1 child2]`